Localization/pt-BR.lang

```diff
@@ -60,361 +60,361 @@
 
 # Misc Items
 #######
-# ItemName.MiceFragment=Mice Fragment
-# ItemTooltip.AncientClickingGlove='Erratic sparks skitter across this fragment'
+# ItemName.MiceFragment=Fragmento de Mouse
+# ItemTooltip.AncientClickingGlove='Faíscas erráticas deslizam por este fragmento'
 
 
 # Accessories
 #######
-# ItemName.AncientClickingGlove=Ancient Clicking Glove
-# ItemTooltip.AncientClickingGlove={$Mods.ClickerClass.Common.Tooltips.ClickingGloveEvery} second
+# ItemName.AncientClickingGlove=Luva de Clicar Antiga
+# ItemTooltip.AncientClickingGlove={$Mods.ClickerClass.Common.Tooltips.ClickingGloveEvery} segundo
 
-# ItemName.ChocolateChip=Chocolate Chip
-# ItemTooltip.ChocolateChip=Every 15 clicks releases a burst of damaging chocolate
+# ItemName.ChocolateChip=Gota de Chocolate
+# ItemTooltip.ChocolateChip=Cada 15 cliques libera uma explosão de chocolate que causa dano
 
-# ItemName.BigRedButton=Big Red Button
-# ItemTooltip.BigRedButton=Every 25 clicks causes a large, fiery explosion
+# ItemName.BigRedButton=Botão Vermelho Grande
+# ItemTooltip.BigRedButton=Cada 25 cliques causa uma explosão flamejante grande
 
-# ItemName.ChocolateMilkCookies=Chocolate Milk n' Cookies
+# ItemName.ChocolateMilkCookies=Leite com Chocolate e Biscoitos
 # ItemTooltip.ChocolateMilkCookies={$Mods.ClickerClass.ItemTooltip.MilkCookies}\n{$Mods.ClickerClass.ItemTooltip.ChocolateChip}
 
-# ItemName.ClickerEmblem=Clicker Emblem
+# ItemName.ClickerEmblem=Emblema do Clicker
 # ItemTooltip.ClickerEmblem=15% {$Mods.ClickerClass.Common.Tooltips.IncreasedClickerDamage}
 
-# ItemName.ClickingGlove=Clicking Glove
-# ItemTooltip.ClickingGlove={$Mods.ClickerClass.Common.Tooltips.ClickingGloveEvery} 3 seconds
+# ItemName.ClickingGlove=Luva de Clicar
+# ItemTooltip.ClickingGlove={$Mods.ClickerClass.Common.Tooltips.ClickingGloveEvery} 3 segundos
 
-# ItemName.Cookie=Cookie
-# ItemTooltip.Cookie=While equipped, cookies will periodically spawn within your clicker radius\nClick the cookie to gain bonus clicker damage, radius, and life regeneration
+# ItemName.Cookie=Biscoito
+# ItemTooltip.Cookie=Enquanto equipado, biscoitos serão gerados periodicamente dentro do seu raio de clique\nClique o biscoito para ganhar Click the cookie to gain bonus clicker damage, radius, and life regeneration
 
```

Comment:
leftover ` Click the cookie to gain bonus clicker damage, radius, and life regeneration` at the end